src/main/java/edu/hawaii/its/api/type/OotbActiveUserProfile.java

```diff
@@ -0,0 +1,91 @@
+package edu.hawaii.its.api.type;
+
+import java.util.List;
+
+public class OotbActiveUserProfile {
+
+    private final String name;
+    private final String uid;
+    private final String uhUuid;
+    private final List<String> authorities;
+    private final String cn;
+    private final String email;
+
+    private OotbActiveUserProfile(Builder builder) {
+        this.name = builder.name;
+        this.uid = builder.uid;
+        this.uhUuid = builder.uhUuid;
+        this.authorities = builder.authorities;
+        this.cn = builder.cn;
+        this.email = builder.email;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public String getUid() {
+        return uid;
+    }
+
+    public String getUhUuid() {
+        return uhUuid;
+    }
+
+    public List<String> getAuthorities() {
+        return authorities;
+    }
+
+    public String getCn() {
+        return cn;
+    }
+
+    public String getEmail() {
+        return email;
+    }
+
+    // Builder class
```

Comment:
One of my guiding principles when coding is to use comments only when needed, especially for messy and complex code that cannot look any more readable. In this case, the `// Builder class` comment is not needed since the code right below, `public static class Builder` says that it is a `Builder` class. Just a small nitpick, I used to comment everything. Now I focus on writing readable code rather than write messy code with comments trying to make up for it.